tensorflow/python/framework/ops.py

```diff
@@ -2240,7 +2240,7 @@ def _add_function(self, function):
 
   @property
   def building_function(self):
-    """Returns True iff this graph represents a function."""
+    """Returns True if this graph represents a function."""
```

Comment:
This doesn't look like a typo.  "iff" stands for "if and only if" here, I think.
https://www.merriam-webster.com/dictionary/iff